programs/vote/src/vote_instruction.rs

```diff
@@ -360,6 +360,12 @@ pub fn process_instruction(
                 &from_keyed_account::<Clock>(keyed_account_at_index(keyed_accounts, 2)?)?,
                 &vote,
                 &signers,
+//                if invoke_context.is_feature_active(&feature_set::vote_credits_timeliness::id()) {
+//                    Some(get_sysvar::<SlotHistory>(invoke_context, &sysvar::slot_history::id())?.next_slot - 1)
```

Comment:
The -1 is cruft, it was since fixed in the uncommented test code, and the -1 will be removed from here too.